### PR TITLE
Long vol in readers

### PR DESCRIPTION
**Pull request by kerautret**
PR Description
Add longvol in various readers.
In progress, todo:
 - [x] 3dImageViewer
 - [x] sliceViewer
 - [x] 3dVolViewer
 - [x] vol2sdp
 - [x] vol2obj
 - [x] vol2slice
 - [x] vol2raw 
 - [x] vol2heightfield
 - [x] volBoundary2obj

# Checklist

- [x] Doxygen documentation of the code completed (classes, methods, types, members...).
- [x] Main tool doxygen documentation (following existing documentation of [DGtalTools documentation](http://dgtal.org/doc/tools/nightly/).
- [x] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools/blob/master/CONTRIBUTING.md)
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools/blob/master/ChangeLog.md) added.
- [x] Update the readme with potentially a screenshot of the tools if it applies. 
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).

### CODE REVIEW

**Comment by dcoeurjo** [comment]
Could yo please update de "-i" description?

I'd need to have a look to the GenericReader with rescale..

**Comment by kerautret** [comment]
Thanks I forget, that done ;)

**Comment by dcoeurjo** [comment]
Do you know why importWithValueFunctor documentation is not visible on Dgtal http://dgtal.org/doc/nightly/structDGtal_1_1GenericReader.html ?

File: visualisation/3dImageViewer.cpp [hunk 1]
```diff
@@ -64,67 +64,70 @@ using namespace Z3i;
...
- @code
-    3dImageViewer -i $DGtal/examples/samples/lobster.vol --thresholdImage -m 180
- @endcode
-
- You should obtain such a result:
-
- @image html res3dImageViewer.png "resulting visualisation of 3d image with thresholded set of voxels."
+   @code
+   -h [ --help ]                    display this message
+   -i [ --input ] arg               vol file (.vol, .longvol .p3d, .pgm3d and if
```
Comment (dcoeurjo) [inline visualisation/3dImageViewer.cpp]: you should add something that the data will be scaled for longvol/dicom/... formats

**Comment by dcoeurjo** [comment]
otherwise it's fine. thanks;)

**Comment by kerautret** [comment]
I thinks it is because it not defined in all specialised template (in particular it is not defined in GenericReader< TContainer, 3, DGtal::uint64_t >DGtal). But perhaps we can add an issue to add a ref to the template specialisation.

File: visualisation/3dImageViewer.cpp (hunk 1 above)
Comment (kerautret) [inline visualisation/3dImageViewer.cpp]: done thanks, I continue with other tools...

File: visualisation/3dImageViewer.cpp [hunk 2]
```diff
@@ -76,7 +76,9 @@ using namespace Z3i;
    -h [ --help ]                    display this message
    -i [ --input ] arg               vol file (.vol, .longvol .p3d, .pgm3d and if
                                     WITH_ITK is selected: dicom, dcm, mha, mhd) 
-                                    or sdp (sequence of discrete points).
+                                    or sdp (sequence of discrete points). For 
+                                    longvol, dicom, dcm, mha or mhd formats, the 
+                                    input values are scaled into 0 and 255.
```
Comment (dcoeurjo) [inline visualisation/3dImageViewer.cpp]: "are linearly scaled between 0 and 255"  ?

Comment (kerautret) [inline visualisation/3dImageViewer.cpp]: yes thanks ;)

**Comment by kerautret** [comment]
@dcoeurjo could we merge ?

File: ChangeLog.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
+# DGtalTools 0.9.4
+
+- *visualisation*:
+  - Improve visualisation tools (3dImageViewer, sliceViewer, 3dVolViewer, vol2sdp)
```
Comment (dcoeurjo) [inline ChangeLog.md]: please update changelog

**Comment by kerautret** [comment]
done thanks ;)

**Comment by dcoeurjo** [comment]
ok thanks. merging

**Pull request merged**